conferences/models.py

```diff
@@ -27,11 +29,28 @@ class Zosia(models.Model):
         default=False
     )
 
-    banner = models.ImageField()
+    banner = models.ImageField(blank=True, null=True)
 
     place = models.ForeignKey(Place)
 
     def __str__(self):
-        return 'In {} from {} to {}'.format(str(self.place),
-                                            self.start_date,
-                                            self.end_date)
+        return 'Zosia {}'.format(self.start_date.year)
+
+    def clean(self):
+        if self.start_date > self.end_date:
+            raise ValidationError(
+                _('Zosia has to have non-negative duration')
+            )
+        super(Zosia, self).clean()
+
+    def validate_unique(self, **kwargs):
+        # NOTE: If this instance is not yet saved, self.pk == None
+        # So this query will take all active objects from db
+        if self.active and Zosia.objects.exclude(pk=self.pk).filter(active=True).exists():
+            raise ValidationError(
+                _('Only one Zosia may be active at any given time')
+            )
+        super(Zosia, self).validate_unique(**kwargs)
+
+    def find_active():
```

Comment:
Może przerzucimy to do managera? Bo w tym momencie potrzebujemy instancję obiektu, żeby z tego skorzystać. 
Brak `self`